app/javascript/tests/enrollment/steps/Contact.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme';
+import Contact from '../../../components/enrollment/steps/Contact.js'
+import { mockPatient1, mockPatient2 } from '../../mocks/mockPatients'
+
+const newEnrollmentState = {
+  isolation: false,
+  patient: {},
+  propagatedFields: {}
+}
+
+const requiredStrings = [
+  'PRIMARY TELEPHONE NUMBER',
+  'SECONDARY TELEPHONE NUMBER',
+  'PRIMARY PHONE TYPE',
+  'SECONDARY PHONE TYPE',
+  'E-MAIL ADDRESS',
+  'CONFIRM E-MAIL ADDRESS'
+]
+
+describe('Monitoree Contact', () => {
+  it('Properly renders all main components', () => {
+    const wrapper = mount(<Contact goto={() => {}} next={() => {}} setEnrollmentState={() => {}} currentState={newEnrollmentState} />);
+    requiredStrings.forEach(requiredString => {
+      expect(wrapper.text().includes(requiredString)).toBe(true);
+    })
+  });
+
+  it('Properly allows setting of all Contact information', () => {
```

Comment:
This only seems to be checking when the preferred contact method is set. If you want to test setting of all contact info, I'd actually test when each of those values change.